modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/GridDiscoveryManager.java

```diff
@@ -1517,20 +1518,40 @@ private long configuredOffheap() {
      */
     private void topologySnapshotMessage(IgniteClosure<String, Void> clo, long topVer, DiscoCache discoCache,
         int evtType, ClusterNode evtNode, int srvNodesNum, int clientNodesNum, int totalCpus, double heap,
-        double offheap) {
+        double offheap, boolean needNodesDetails) {
```

Comment:
The method's signature violates our code style. Please place each parameter on a separate line. Look at {{GridDiscoveryManager#fillAffinityNodeCaches}} method as a reference.

Reply:
I read Coding Guidelines. There are two styles of method declaration.
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines
If arguments can't fit to single line, arguments are splitted by 120 char limit to several lines.